sass/layout/_headers.scss

```diff
@@ -269,6 +269,7 @@
             .emoticon {
                 position: relative;
                 top: -1px;
+                overflow: hidden;
```

Comment:
I didn't know the header had its own styling for emojis. I'm not at all surprised this broke then